src/main/java/org/openhab/tools/analysis/checkstyle/BundleVendorCheck.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
Yes, some of the other classes need to be updated, too. :) 
Thanks for noticing.